cmd/stack_select.go

```diff
@@ -86,5 +97,13 @@ func newStackSelectCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVarP(
 		&stack, "stack", "s", "",
 		"The name of the stack to select")
+	cmd.PersistentFlags().BoolVarP(
+		&create, "create", "c", false,
+		"If selected stack does not exist, create it")
+	cmd.PersistentFlags().StringVar(
+		&secretsProvider, "secrets-provider", "default",
+		"Use with --create flag, "+
+			"the type of the provider that should be used to encrypt and decrypt secrets"+
+			"(possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault)")
```

Comment:
Sorry for the tedious suggestion, but this is going to be difficult to maintain whenever we support a new secrets provider. (Since I assume this was copied directly from the code for `stack init`?)

We should instead declare a `const possibleSecretsProviderChoices = "(possible choices: ...)"` value, and use that in both places. So that everything will "just work", and we won't introduce any inconsistencies as the code evolves.